Create property that determines if external IDs are shown inside location details modal (#377)
MapsPeople/web-ui: packages/map-template/CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.51.0] - 2024-06-17

### Added

> - Add support for new parameter `showExternalIDs` which determines if selected location should have externalId visible.

Reviewer: ```suggestion
- Add support for new parameter `showExternalIDs` which determines the Location details should show the External ID or not.
```
